diff --git a/node_modules/@mui/material/InputBase/InputBase.d.ts b/node_modules/@mui/material/InputBase/InputBase.d.ts
index fca9a60..1c8bfeb 100644
--- a/node_modules/@mui/material/InputBase/InputBase.d.ts
+++ b/node_modules/@mui/material/InputBase/InputBase.d.ts
@@ -102,7 +102,9 @@ export interface InputBaseProps extends StandardProps<React.HTMLAttributes<HTMLD
    * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input#attributes) applied to the `input` element.
    * @default {}
    */
-  inputProps?: InputBaseComponentProps;
+  inputProps: {
+    'data-testid': string;
+  } & InputBaseComponentProps;
   /**
    * Pass a ref to the `input` element.
    */
diff --git a/node_modules/@mui/material/esm/InputBase/InputBase.d.ts b/node_modules/@mui/material/esm/InputBase/InputBase.d.ts
index fca9a60..1c8bfeb 100644
--- a/node_modules/@mui/material/esm/InputBase/InputBase.d.ts
+++ b/node_modules/@mui/material/esm/InputBase/InputBase.d.ts
@@ -102,7 +102,9 @@ export interface InputBaseProps extends StandardProps<React.HTMLAttributes<HTMLD
    * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input#attributes) applied to the `input` element.
    * @default {}
    */
-  inputProps?: InputBaseComponentProps;
+  inputProps: {
+    'data-testid': string;
+  } & InputBaseComponentProps;
   /**
    * Pass a ref to the `input` element.
    */
