diff --git a/node_modules/@mui/material/InputBase/InputBase.d.ts b/node_modules/@mui/material/InputBase/InputBase.d.ts
index 311d053..bc51b62 100644
--- a/node_modules/@mui/material/InputBase/InputBase.d.ts
+++ b/node_modules/@mui/material/InputBase/InputBase.d.ts
@@ -80,7 +80,9 @@ export interface InputBaseProps extends StandardProps<React.HTMLAttributes<HTMLD
    * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input#attributes) applied to the `input` element.
    * @default {}
    */
-  inputProps?: InputBaseComponentProps | undefined;
+  inputProps: {
+    'data-testid': string;
+  } & InputBaseComponentProps;
   /**
    * Pass a ref to the `input` element.
    */
